src/cadence/mainnet/getNFTs.cdc

```diff
@@ -1248,4 +1250,45 @@ pub fun getXtinglesNFT(owner: PublicAccount, id: UInt64): NFTData? {
             "edition": nft!.metadata!.edition   
         },
     )
-}
\ No newline at end of file
+}
+
+// https://flow-view-source.com/mainnet/account/0x6831760534292098/contract/OneFootballCollectible
+pub fun getOneFootballCollectible(owner: PublicAccount, id: UInt64): NFTData? {
+    let contract = NFTContract(
+        name: "OneFootballCollectible",
+        address: 0x6831760534292098,
+        storage_path: "OneFootballCollectible.CollectionStoragePath",
+        public_path: "OneFootballCollectible.CollectionPublicPath",
+        public_collection_name: "OneFootballCollectible.OneFootballCollectibleCollectionPublic",
+        external_domain: "http://xmas.onefootball.com/"
+    )
+
+    if let collection = owner
+        .getCapability<&OneFootballCollectible.Collection{OneFootballCollectible.OneFootballCollectibleCollectionPublic}>(OneFootballCollectible.CollectionPublicPath)
+            .borrow() {
+        if let nft = collection.borrowOneFootballCollectible(id: id) {
+            if let metadata = nft.getTemplate() {
+                return NFTData(
+                    contract: contract,
+                    id: nft.id,
+                    uuid: nft.uuid,
+                    title: metadata.name,
+                    description: metadata.description,
+                    external_domain_view_url: "https://xmas.onefootball.com/".concat(owner.address.toString()),
+                    token_uri: nil,
```

Comment:
I'm not sure what this line is for as the Readme doesn't say :/ let me know if I can add anything here @sideninja @rambhask @niveda-krish

Reply:
Token_uri can be used if your metadata is hosted on ipfs/arweave (similar to the way its used in Eth).